### DISCUSSION TITLE
feature: Make it possible to set location of notifications/messages

### DISCUSSION THREAD

**Discussion by kimusan** [folke/noice.nvim#835] [Q&A]
### Did you check the docs?

- [X] I have read all the noice.nvim docs

### Is your feature request related to a problem? Please describe.

I have tried finding a way to place the notifications in the lower right corner instead og top right. I have not been able to find a configuration for this. I would like to be able tell Noice where the different parts (notifications, messages, etc) should be shown.

### Describe the solution you'd like

make it possible to configure the position for things like e.g. 

position = "middle,right"

Vertical position and horizontal position. 

### Describe alternatives you've considered

none yes

**Comment by b0ae989c** [comment]
Could you share your `noice` config? I use `mini` for message UI backend and I have them in the lower right corner. My config is
```lua
messages = { view = 'mini' },
views = {
  mini = {
    position = { col = -1, row = -1 },
  },
},
```

**Comment by folke** [comment]
Don't use issues for user questions. That's what discussions are for. It's even in the issue template
